src/functions-templates/typescript/scheduled-function/{{name}}.ts

```diff
@@ -0,0 +1,15 @@
+import { Handler, schedule } from '@netlify/functions';
+
+const formatAsDateTime = (date: Date) => `${date.toLocaleDateString()} ${date.toLocaleTimeString()}`;
+
+const handler: Handler = () => {
+    const currentExecutionTime = formatAsDateTime(new Date(Date.now()));
+    console.log(`Function executed at ${currentExecutionTime}.`);
+
+    return {
+        statusCode: 200
+    };
+};
+
+module.exports.handler = schedule("* * * * *", handler);
```

Comment:
[boulder] We should use [ESM exports](https://developer.mozilla.org/en-US/docs/web/javascript/reference/statements/export) in TypeScript. See the [other TypeScript templates](https://github.com/netlify/cli/blob/main/src/functions-templates/typescript/hello-world/%7B%7Bname%7D%7D.ts).

Reply:
Lazy copying at work here 🙊